lego/apps/content/models.py

```diff
@@ -25,7 +25,7 @@ class Meta:
 
     def generate_slug(self):
```

Comment:
*Not related to this code specifically, I just put it here so I could have a thread(:*

Has there been any discussion about the advantages of this approach vs just adding a "slug-ish" version of the event title to the URL? Such as `https://abakus.no/events/3337` -> `https://abakus.no/events/3337-abacup` or `https://abakus.no/events/3337/abacup` where the slug is ignored and only the id is queried?

Personally I'm partial towards the latter approach, as the slugs don't become outdated if the organizers decide to change the name of the event + we don't end up with dead links if we change the slug later because the id is dominant.

Reply:
There is some discussion on [ABA-210](https://linear.app/abakus-webkom/issue/ABA-210/add-slug-to-events-joblistings-interestgroups).
I started out with /{id}/{slug} and just querying the id, but just the slug is prettier I think. 
We could overengineer this properly by changing the slug when title is changed, and also storing all the slugs in the db for query, but this might have performance issues. 

Also, I don't know quite how often event names are changed, but we could just try this approach and if people complain about outdated slugs/they become prevalent, we can address the issue then. At least if slugs never changed after creation, there will be no dead links.